website/static/docs/accordproject-faq.md

```diff
@@ -0,0 +1,58 @@
+---
```

Comment:
What's the difference between the `./website/static/docs` directory and `./docs`, please? I assume that we only need one of them?

Reply:
We need docs in static folder because the line ```![Template-Studio-V2](/docs/assets/reference/tsv2.png)``` in ```website/versioned_docs/version-0.21/ref-cicero-ui.md``` is looking for these files in ``website/static/docs/assets/reference/tsv2.png``` thus leading to redundancy. Any way we can address this problem by changing the folder where it looks for files to direct to our docs folder ?

Reply:
Some versioned files had links like ```/docs/assets/*.png```, they were searching for these files in static folder and causing errors so earlier I copied the docs folder into static folder. Now, in adee0af8e3bbe28a853118d979b5175da5be2c1d I've fixed this issue, removing unnecessary website/docs/*  and website/static/docs/*.md while preserving website/static/docs/assets.